nistats/reporting/reporting_examples/report_flm_adhd_dmn/report_flm_adhd_dmn.py

```diff
@@ -0,0 +1,61 @@
+import numpy as np
```

Comment:
Cuz then sphinx generates an error cuz they are not documented properly and bla bla. SInce these may not even be the final examples we use, I don't want to spend a ton of time dealing with rst only to delete them. Once we have agreement on how to showcase the functionality, we can move them then.

Reply:
ok, no problem